modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,842 @@
+----
+SHOW DATABASE `northwind-graph-2021`
+----
+
+.Result
+[role="queryresult"]
+----
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name                   | type       | aliases       | access       | address          | role      | writer | requestedStatus | currentStatus | statusMessage | default | home  | constituents |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "northwind-graph-2021" | "standard" | ["northwind"] | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | FALSE   | FALSE | []           |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+
+=== Alter remote database aliases
+
+Example of altering a remote database alias target.
+
+.Query
+[source, cypher]
+----
+ALTER ALIAS `remote-northwind` SET DATABASE
+TARGET `northwind-graph-2020` AT "neo4j+s://other-location:7687"
+----
+
+=== Alter remote credentials and driver settings for remote database aliases
+
+Example of altering a remote database alias credentials and driver settings.
+
+.Query
+[source, cypher]
+----
+ALTER ALIAS `remote-with-driver-settings` SET DATABASE
+USER bob
+PASSWORD 'new_example_secret'
+DRIVER {
+  connection_timeout: duration({ minutes: 1}),
+  logging_level: 'debug'
+}
+----
+
+[IMPORTANT]
+====
+All driver settings are replaced by the new ones.
+In this case, by not repeating the driver setting `connection_pool_max_size` the value will be deleted and fall back to the default value.
+====
+
+=== Remove custom driver settings from remote database aliases
+
+Example of altering a remote database alias to remove all custom driver settings.
+
+.Query
+[source, cypher]
+----
+ALTER ALIAS `movie scripts` SET DATABASE
+DRIVER {}
+----
+
+=== Alter properties for local and remote database aliases
+
+Examples of altering local and remote database alias properties.
+
+.Query
+[source, cypher]
+----
+ALTER ALIAS `motion pictures` SET DATABASE PROPERTIES { nameContainsSpace: true, moreInfo: 'no, not really' }
+----
+
+.Query
+[source, cypher]
+----
+ALTER ALIAS `movie scripts` SET DATABASE PROPERTIES { nameContainsSpace: true }
+----
+
+The updated properties can then be used in queries with the xref::database-administration/composite-databases/querying-composite-databases.adoc#composite-databases-queries-built-in-functions[`graph.propertiesByName()` function].
+
+=== Use `IF EXISTS` when altering database aliases
+
+
+The `ALTER ALIAS` command is optionally idempotent, with the default behavior to fail with an error if the database alias does not exist.
```

Comment:
since this comment lives down in this section for alter, should the corresponding section for create also move down into it's section around creating aliases with same name as existing aliases?

Reply:
(mirroring how it was done for create database as well)

Reply:
Have updated